interface/main/calendar/find_appt_popup.php

```diff
@@ -296,7 +280,7 @@ function doOneDay($catid, $udate, $starttime, $duration, $prefcatid)
 
  function setappt(year,mon,mday,hours,minutes) {
   if (opener.closed || ! opener.setappt)
-   alert('<?php echo xls('The destination form was closed; I cannot act on your selection.'); ?>');
+   alert(<?php echo xlj('The destination form was closed; I cannot act on your selection.'); ?>);
```

Comment:
hi @attritionorg , This is a better function for escaping (xss) within javascript code. It is using json_encode()